infrastructure/code/template/details.html

```diff
@@ -366,6 +416,10 @@ <h2 class="severity-title">{{.IssueTitle}}</h2>
       {{if ne $index (idxMinusOne (len $.CWEs))}}<span class="delimiter"></span>{{end}}
       {{end}}
       {{end}}
+      {{if .IgnoreLineAction}}
+      <span class="delimiter"></span>
+      <div class="priority-score">Position line: {{.IgnoreLineAction}}</div>
```

Comment:
why class ="priority-score"

Reply:
It can be deleted in the continues iterations - Thank you!